libafl_qemu/src/injections.rs

```diff
@@ -334,6 +334,7 @@ extern "C" fn on_call_check(val: u64, _pc: GuestAddr) {
         let injection = &vec[off];
         //println!("Checking {}", injection.name);
         for test in &injection.tests {
+            #[allow(clippy::manual_assert)]
```

Comment:
Why not just use an assert?

Reply:
Also @andreafioraldi @rmalmain could we exit from the fuzzing loop here without crashing?